charts/models/values.yaml

```diff
@@ -408,6 +408,12 @@ catalog:
       - --enforce-eager
     engine: VLLM
     resourceProfile: 'nvidia-gpu-l4:1'
+  deepseek-r1-1.5b-ollama:
```

Comment:
Current convention is to suffix with `-cpu`. I would prefer to keep with that, or refactor all model names.

Reply:
done